### ISSUE TITLE
Update README for Improved Documentation

### ISSUE THREAD

**Issue by jinx-vi-0** [MAVRICK-1/e-commerce_website#75]
Clear and well-structured documentation is crucial for users and contributors to understand the project, its purpose, and how to get started. The current README is not visually appealing and also lacks some begineer friendly contribution start guide.

hey @MAVRICK-1 , please assign this issue to me.

**Comment by MAVRICK-1** [comment]
@jinx-vi-0 we got learn.md for that

**Comment by jinx-vi-0** [comment]
@MAVRICK-1 , so I can make README more visually appealing by adding badges for tech stack and other proper formatting of the README.

**Comment by MAVRICK-1** [comment]
@jinx-vi-0 okay go ahead

**Comment by MAVRICK-1** [comment]
@jinx-vi-0 i am giving level 2 for now , if i like your work it will be upgraded to level 3

**Comment by jinx-vi-0** [comment]
ok will sure impress you

**Comment by MAVRICK-1** [comment]
@jinx-vi-0 can you fix issue with contributors section in Readme.md it is not show images of all the contributors

**Comment by jinx-vi-0** [comment]
@MAVRICK-1 ya sure.

**Comment by jinx-vi-0** [comment]
> @jinx-vi-0 can you fix issue with contributors section in Readme.md it is not show images of all the contributors

I've looked into the issue with the contributors section. It appears that not all contributors' images are being displayed here, although they are visible on the website. After some investigation, I've discovered that this is a known issue with the website. It caches the initial request, causing subsequent requests to be queued. The display should auto-update later to include all contributors.
